apps/balancer-tools/src/components/SearchPoolForm.tsx

```diff
@@ -88,7 +94,7 @@ export function SearchPoolForm({
   const filteredPoolList = poolsDataList?.pools
     .filter((pool) => filterPoolInput({ poolSearchQuery: poolId, pool }))
     .sort((a, b) =>
-      Number(a!.totalLiquidity) < Number(b!.totalLiquidity) ? 1 : -1,
+      Number(a!.totalLiquidity) < Number(b!.totalLiquidity) ? 1 : -1
```

Comment:
👀 this isn't in line with our prettier -- can you make sure your VSCode is sourcing the config correctly?